config/routes.rb

```diff
@@ -9,6 +9,10 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :create, :show]
+  end
+  resources :posts do
+    resources :comments, only: [:create]
```

Comment:
[REQUIRED] Please don't forget to include routes for the new action after creating it.